manifests/init.pp

```diff
@@ -62,7 +62,17 @@
 
   file { '/etc/condor/config.d/schedd':
     ensure  => file,
-    content => 'DAEMON_LIST = MASTER, SCHEDD',
+    content => @(CONTENT)
+      DAEMON_LIST = MASTER, SCHEDD
+      HISTORY_CONTAINS_JOB_ENVIRONMENT = False
+      | CONTENT
+    ,
+    require => Yumrepo['condor'],
```

Comment:
Why does this resource depend on a yumrepo being configured?

Reply:
it should be a package been installed, not a yumrepo... no packages, no directory... thanks!